src/main/jbake/content/documentation/arc42/src/01_introduction_and_goals.adoc

```diff
@@ -1,112 +1,109 @@
 [[section-introduction-and-goals]]
 == Introduction and Goals
 
-Partner dance is something great from many perspectives:
+Partner dance is great from many perspectives:
 
-* it brings people together _(social perspective)_
-* dancing is a good full-body workout _(physical health perspective)_
-* dancing relieves stress _(mental health perspective)_
-* dancing people in general seem to be less interested in starting wars _(society's perspective)_
-* not first but: you can find a partner for life _(mankind's survival perspective)_
+* It brings people together _(social perspective)_
+* Dancing is a good full-body workout _(physical health perspective)_
+* Dancing relieves stress _(mental health perspective)_
+* In general, people who dance seem to be less interested in starting wars _(society's perspective)_
+* Last but not least, you can find a partner for life _(mankind's survival perspective)_
 
 *BUT!*
 
-There is quite often one barrier for one starting with a couple dance. You have to find a partner to dance with.
+There is often one barrier for someone starting with a partner dance. You have to find a partner to dance with.
 
 Dancier is there to help all those people!
 
 
 === Functional Requirements
 
-Dancier is going to be the go-to platform for all people who dance or want to dance. It is not limited to a certain kind of dance like Tango or Salsa.
+Dancier aims to be the go-to platform for dance enthusiasts of all types, regardless of their preferred dance style, be it Tango, Salsa, or any other.
 
-Although in the beginning, Dancier will focus its features on couple dances. So all dance styles where two people dance with each other.
+In its initial stage, our primary focus with our list of requirements will center around partner dances, encompassing a wide array of styles where two individuals dance together.
+As Dancier evolves into a platform where people can find dance partners, we will consider adding more features, such as enabling dance schools to provide information about their courses and dance events.
 
-So as of now, we will first define the requirements that are needed to bring two people together.
-
-When Dancier is a platform where people can find dancing partners, we will add functionality like enabling Dance Schools to provide information on their courses and dance evenings...
-
-For this reason, in the following table, we also give information on what is needed for the https://en.wikipedia.org/wiki/Minimum_viable_product[MVP] as mandatory and the as optional.
+Based on that, we provide details on what is required for our [Minimum Viable Product (MVP)](https://en.wikipedia.org/wiki/Minimum_viable_product) and what can be considered optional in the following table.
```

Comment:
scheinbar hab ich den Link hier kaputt gemacht

Reply:
Du oder ChatGPT ;-) I will fix this.